integration/run_test.go

```diff
@@ -231,3 +233,83 @@ func TestFix(t *testing.T) {
 		t.Fatalf("testing error: %v", err)
 	}
 }
+
+func TestInit(t *testing.T) {
+	type testCase struct {
+		name string
+		dir  string
+		args []string
+	}
+
+	tests := []testCase{
+		{
+			name: "getting-started",
+			dir:  "../examples/getting-started",
+		},
+		{
+			name: "microservices",
+			dir:  "../examples/microservices",
+			args: []string{
+				"-a", "leeroy-app/Dockerfile=gcr.io/k8s-skaffold/leeroy-app",
+				"-a", "leeroy-web/Dockerfile=gcr.io/k8s-skaffold/leeroy-web",
+			},
+		},
+	}
+
+	for _, test := range tests {
+		t.Run(test.name, func(t *testing.T) {
```

Comment:
if you do `skaffold run -f -` you can pipe in the skaffold yaml over stdin!

Reply:
if only I had been a clever boy, I would have done this